roles/ceph-validate/tasks/check_system.yml

```diff
@@ -78,10 +78,14 @@
       when: ansible_distribution not in ['RedHat', 'CentOS', 'Fedora']
 
     - name: fail on unsupported distribution version for iscsi gateways
-      fail:
-        msg: "iSCSI gateways can only be deployed on Red Hat Enterprise Linux or CentOS >= 7.4"
-      when:
-        - ansible_distribution_version < '7.4'
-        - ansible_distribution in ['RedHat', 'CentOS']
+      command: 'grep -q {{ item }}=m /boot/config-{{ ansible_kernel }}'
```

Comment:
this file isn't always present (atomic host):

```bash
-bash-4.2# ls /boot/
efi  grub  grub2  loader  loader.0  ostree
```

Reply:
Yep I saw that last friday...